code/modules/mob/living/simple_animal/abnormality/waw/pygmalion.dm

```diff
@@ -128,6 +128,10 @@
 	remove_status_effect(STATUS_EFFECT_SCULPTOR)
 	threat_level = WAW_LEVEL
 	sculptor = null
+	if (sculptor)
+		sculptor.remove_status_effect(STATUS_EFFECT_SCULPTOR)
+	if (missing_prudence)
+		restorePrudence()
```

Comment:
None of this works because `sculptor = null` is before this.

Reply:
fixed. Thank you!